rest-api-spec/src/main/resources/rest-api-spec/test/cat.indices/10_basic.yml

```diff
@@ -8,6 +8,9 @@
       $body: |
                /^$/
 
+---
+"Test cat indices output":
+
```

Comment:
This test is split because replicated closed indices can report docs stats while non-replicated closes indices can't